src/app/modules/form/public-form/public-form.controller.ts

```diff
@@ -485,33 +486,40 @@ export const handleFormAuthRedirect = [
 
 /**
  * NOTE: This is exported only for testing
- * Logs user out of SP / CP By deleting cookie
+ * Logs user out of SP / CP / MyInfo / SGID by deleting cookie
  * @param authType type of authentication
  *
  * @returns 200 with success message when user logs out successfully
  * @returns 400 if authType is invalid
  */
 export const _handleSpcpLogout: ControllerHandler<
-  { authType: AuthType.SP | AuthType.CP | AuthType.SGID },
+  { authType: AuthType.SP | AuthType.CP | AuthType.MyInfo | AuthType.SGID },
   PublicFormAuthLogoutDto
 > = (req, res) => {
   const { authType } = req.params
 
+  const cookieName =
+    authType === AuthType.MyInfo
+      ? MYINFO_COOKIE_NAME
+      : authType === AuthType.SGID
+      ? SGID_COOKIE_NAME
+      : JwtName[authType]
```

Comment:
ok for now but I generally don't favour nested ternary operators. hard to read and parse the logic. I would prefer it if we use `let` with a `switch` statement, or just repeat the `res.clearCookie` statement as necessary

Reply:
not keen on `let` so I went with `if` + repeat, see if ok